app/uk/gov/hmrc/testuser/models/Payloads.scala

```diff
@@ -16,16 +16,21 @@
 
 package uk.gov.hmrc.testuser.models
 
+case class AuthenticationRequest(username: String, password: String)
 
 object ErrorCode extends Enumeration {
 
   type ErrorCode = Value
 
   val INTERNAL_SERVER_ERROR = Value("INTERNAL_SERVER_ERROR")
+  val WRONG_PASSWORD = Value("WRONG_PASSWORD")
+  val USERNAME_NOT_FOUND = Value("USERNAME_NOT_FOUND")
```

Comment:
@tom59: Because these are test users, I thought it would be better to be more specific.  

_However_, we do not have any immediate plans to disclose this via any frontend, so I would not mind using the standard `INVALID_CREDENTIALS` approach (so long as we also log the failure + reason for it locally to this service). 

It would be a better application of YAGNI to do what you suggest.  

@gokyo : If you don't mind updating what you've done, I'm happy to update the ticket.